package.json

```diff
@@ -16,7 +16,10 @@
     "test.ci": "jest --maxWorkers=2",
     "test.coverage": "yarn test --coverage",
     "test.watch": "yarn test --watchAll",
+    "run.binary": "cd cli-binaries && ./$BINARY mock ../examples/$SPEC",
     "test.update": "yarn test.fast --updateSnapshot",
+    "build.binaries": "yarn && yarn build && npx pkg --out-path ./cli-binaries ./packages/cli/",
```

Comment:
Use pre scripts instead of multiple commands. Also, I think you can assume the basic yarn command has been ran already.